build/Makefile

```diff
@@ -363,9 +368,12 @@ install: $(ensure-build-image) install-custom-pull-secret
 		--set agones.image.tag=$(VERSION),agones.image.registry=$(REGISTRY) \
 		--set agones.image.controller.pullPolicy=$(IMAGE_PULL_POLICY),agones.image.sdk.alwaysPull=$(ALWAYS_PULL_SIDECAR) \
 		--set agones.image.controller.pullSecret=$(IMAGE_PULL_SECRET) \
+		--set agones.image.extensions.pullPolicy=$(IMAGE_PULL_POLICY) \
```

Comment:
Don't set these just yet, since we're not using them.

Reply:
Done, removed them.